spec/slack-ruby-bot/commands/unknown_spec.rb

```diff
@@ -4,9 +4,16 @@
   def app
     SlackRubyBot::App.new
   end
+
+  def unknown_response
```

Comment:
No need to refactor this one out, some duplication in tests for messages is ok :)